src/block/accordion/index.php

```diff
@@ -18,3 +18,35 @@ function stackable_load_accordion_frontend_script() {
 	}
 	add_action( 'stackable/accordion/enqueue_scripts', 'stackable_load_accordion_frontend_script' );
 }
+
+if ( ! function_exists( 'stackable_load_accordion_frontend_polyfill_script' ) ) {
+	require_once( ABSPATH . 'wp-admin/includes/dashboard.php' );
```

Comment:
What is this `require_once` for? If this is needed, maybe we should add this inside the area / condition where this is used. With this current location, this `require_once` is always present per page load.